qcodes/tests/parameter/test_parameter_registration.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from qcodes.instrument.parameter import Parameter
+from qcodes.tests.instrument_mocks import DummyAttrInstrument
+from qcodes.utils.deprecate import QCoDeSDeprecationWarning
+
+
+class BrokenParameter(Parameter):
+    """A parameter that incorrectly does not forward instrument to the base parameter"""
```

Comment:
```suggestion
    """
    A parameter that incorrectly sets instrument to the super-class parameter,
    instead, it should forward it via ``instrument`` argument
    """
```

Reply:
Imho a bit over the top for a test helper class but sure why not ;)